src/runtime/x86_cpu_features.cpp

```diff
@@ -9,8 +9,9 @@ extern "C" void x86_cpuid_halide(int32_t *);
 
 namespace {
 
-ALWAYS_INLINE void cpuid(int32_t fn_id, int32_t *info) {
+ALWAYS_INLINE void cpuid(int32_t *info, int32_t fn_id, int32_t extra = 0) {
```

Comment:
(1) this changes the argument ordering, so now it's different from the (compile-time) implementation of cpuid in src/Target.cpp; it would be nice to keep them the same to avoid confusion for people reading the code. (I don't really care which ordering is 'correct', just that they match.)
(2) Why is `extra` an argument when it is never set to anything but the default value?

Reply:
The changes here make the parameter order match with `cpuid` in [src/Target.cpp](https://github.com/halide/Halide/blob/f9413766bf227b05429a3ba5e97251df3a431a74/src/Target.cpp#L45). Is there another version of this function with the opposite ordering?

The `extra` parameter is not used now, but is needed to check AVX512BF16 once #5677 is merged. For that check we need to be able to call `cpuid(info, 7, 1)`.

Reply:
...ack. You're right. I misread the change direction. LGTM.